tests/driver.spec.js

```diff
@@ -0,0 +1,19 @@
+const { test, expect } = require('@playwright/test');
+const Driver = require('../pages/Actions/driver'); 
+const { login } = require('../helper/auth');
+
+test.beforeEach(async ({ page }) => {
```

Comment:
If login is the pre-requisite for each and every test, why it is not included in One time setup or any other alternate to remove this repetition